actions/build/expansion.py

```diff
@@ -10,25 +10,22 @@ def __init__(self, ai):
     async def should_handle(self, iteration):
         """Good for now, maybe the 7th or more hatchery can be postponed
          for when extra mining patches or production are needed """
+        base_amount = len(self.ai.townhalls)  # so it just calculate once per loop
+        if not self.worker_to_first_base and base_amount < 2 and self.ai.minerals > 225:
+            self.ai.actions.append(await self.send_worker_to_next_expansion())
```

Comment:
wrong place to append actions, it should happen in the `handle` method of an action

Reply:
my try to fix double worker bug